### PR TITLE
fixed categorical market

### PR DESCRIPTION
**Pull request by bthaile**
commented out bad merge or just not fully flushed out code that was causing errors. 

Fixed categorical market creation so we can test market creation

### CODE REVIEW

**Comment by justinbarry** [comment]
Seeing this error when trying to create a categorical market:
image

**Comment by bthaile** [comment]
Saw that error as well, started using portis to create markets. Will keep investigating meta mask issue

**Review by JohnDanz — DISMISSED** [review]

File: packages/augur-ui/src/modules/account/selectors/core-stats.ts [hunk 1]
```diff
@@ -14,17 +14,14 @@ export const selectCoreStats = createSelector(
       availableFunds: {
         label: "Available Funds",
         value: formatEther(accountFunds.totalAvailableTradingBalance).formatted,
-        mobileValue: MobileLimitNumber(formatEther(accountFunds.totalAvailableTradingBalance).formatted)
```
Comment (JohnDanz) [inline packages/augur-ui/src/modules/account/selectors/core-stats.ts:17]: put this stuff back in please

File: packages/augur-ui/src/modules/account/selectors/core-stats.ts [hunk 2]
```diff
@@ -33,13 +30,3 @@ export const selectCoreStats = createSelector(
     }),
 );
 
-export const MobileLimitNumber = (formattedValue: string) => {
```
Comment (JohnDanz) [inline packages/augur-ui/src/modules/account/selectors/core-stats.ts:36]: and this obviously

File: packages/augur-ui/src/modules/common/labels.tsx [hunk 3]
```diff
@@ -79,7 +79,6 @@ export interface PropertyLabelProps {
 export interface LinearPropertyLabelProps {
   label: string;
   value: string | FormattedNumber;
-  mobileValue?: string | FormattedNumber;
```
Comment (JohnDanz) [inline packages/augur-ui/src/modules/common/labels.tsx:82]: and this whole page

**Review by JohnDanz — APPROVED** [review]

**Pull request merged**